src/sp_config.c

```diff
@@ -61,6 +62,26 @@ int parse_empty(char *restrict line, char *restrict keyword, void *retval) {
   return 0;
 }
 
+int parse_list(char *restrict line, char *restrict keyword, void *list_ptr) {
```

Comment:
Why a `void*` instead of a `sp_list*`?

Reply:
`parse_list` is made to be used in the `sp_config_function`, in the parser, so it should follow ` int (*)(char*,char*,void*)`.